queries.sql

```diff
@@ -1,3 +1,26 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon';
+SELECT name
+FROM animals
+WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT name
+FROM animals
+WHERE neutered = true and escape_attempts >= 3;
```

Comment:
- [ ] Over here, we want to _list the name of all animals that are neutered and **have less than 3** escape attempts_, therefore our query should be similar to:

```sql
  SELECT name FROM animals WHERE neutered = 'true' AND escape_attempts < '3';
  ```